### DISCUSSION TITLE
Is there a guide on how to deploy Milvus in distributed clusters?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#18609] [Q&A and General discussion]
Is there a guide on how to deploy Milvus in distributed clusters?
How do I deploy it on a Kubernetes or just several machines? 

Can someone share his own experience?
Where should I start? 

How do I configure number of pods and instances with helm?

**Comment by yhmo** [comment]
The official document: https://milvus.io/docs/v2.1.x/install_cluster-helm.md

**Reply by martinenkoEduard** [reply]
Yep I have seen it, but it is really concise :)
I mean how to change number of pods, ensure data persistence and so on.

**Reply by martinenkoEduard** [reply]
maybe can someone share his onw experience?
